test/selenium/testlib_avocado/seleniumlib.py

```diff
@@ -231,14 +231,16 @@ def click(self, element):
                     element.click()
                 failure = None
                 break
-            except Exception as e:
+            except WebDriverException as e:
                 failure = e
                 pass
             try:
                 element = usedfunction() if usedfunction else element
                 self.everything_loaded(element)
-            except Exception:
+            except WebDriverException:
                 pass
+            time.sleep(0.2)
```

Comment:
I suggest to not hardcoded it there, but to some constants or as an class attribute.

Reply:
Done.